soccer/gameplay/plays/testing/test_coordinated_pass.py

```diff
@@ -36,9 +36,11 @@ def execute_running(self):
         pass_bhvr = self.subbehavior_with_name('pass')
 
         if pass_bhvr.is_done_running():
-            pass_bhvr.restart()
+            self.remove_subbehavior('pass')
```

Comment:
Whoops, I think this was from some old experimentation... I'll put it back the way it was.